cmd/svcat/binding/unbind_cmd.go

```diff
@@ -131,16 +109,27 @@ func (c *unbindCmd) unbindInstance() error {
 	return nil
 }
 
+func (c *unbindCmd) getBindingsToDelete() []types.NamespacedName {
+	bindings := []types.NamespacedName{}
+	for _, name := range c.bindingNames {
+		bindings = append(bindings, types.NamespacedName{Namespace: c.Namespace, Name: name})
+	}
+	return bindings
+}
+
 // waitForBindingDeletes waits for the bindings to be deleted and prints either
 // and error message or the name of the deleted binding.
-func (c *unbindCmd) waitForBindingDeletes(waitMessage string, bindings ...v1beta1.ServiceBinding) bool {
+func (c *unbindCmd) waitForBindingDeletes(waitMessage string, bindings ...types.NamespacedName) bool {
 	if len(bindings) == 0 {
 		return false
 	}
 
 	// Indicates an error occurred and that a non-zero exit code should be used
 	var hasErrors bool
 
+	// Used to prevent concurrent writes to c.Output
+	var mutex sync.Mutex
```

Comment:
Good catch! I had forgotten that Output is a raw Writer and not a Logger. I may circle back after this goes in to switch so that it's safe to pass around and use in goroutines.